tests/chaos/oadp/conftest.py

```
@@ -79,3 +82,117 @@ def drain_vm_source_node(admin_client, rhel_vm_with_dv_running, oadp_backup_in_p
+    # Cleanup all created VMs
+    for vm, vm_generator in created_vms:
+        try:
+            vm.clean_up()
+        finally:
+            try:
+                vm_generator.__exit__(None, None, None)  # noqa: FCN001
+            except Exception:
+                LOGGER.exception(f"Failed to cleanup VM {vm.name}")
+
+
+@pytest.fixture()
```

Ruff (0.14.14)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)